File: LetQuiz.Domain/Result.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Diagnostics.CodeAnalysis;

namespace LetQuiz.Domain
{
    public class Result
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ResultId { get; set; }

        public float? Point { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime FinishTime { get; set; }

        public int QuizId { get; set; }
        public Quiz Quiz { get; set; }

        [Required]
        public string AccountUsername { get; set; }
        public Account Account { get; set; }

        public List<Answer> Answers { get; set; }
    }
}

File: Let Quiz/ClientApp/src/app/models/result.ts
import {Answer} from './Answer';
import {Quiz} from './Quiz';

export interface Result {
    resultId: number;
    point: number;
    startTime: Date;
    finishTime: Date;
    quizId: number;
    nameOfQuiz: string;
    quiz: Quiz;
    answerDtos: Answer[];
}
